code/API_definitions/device-status.yaml

```diff
@@ -131,50 +153,98 @@ paths:
           $ref: "#/components/responses/Generic500"
         503:
           $ref: "#/components/responses/Generic503"
-  /event-subscriptions:
+  /subscriptions:
     post:
       tags:
-        - Device status event subscription
-      summary: 'Create a device status event subscription for a device'
+        - Device status subscription
+      summary: "Create a device status event subscription for a device"
       description: Create a device status event subscription for a device
-      operationId: createDevicStatusEventSubscription
+      operationId: createDeviceStatusSubscription
+      security:
+        - threeLegged:
+            - device-status:roaming-subscriptions:create
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/CreateEventSubscription'
+              $ref: "#/components/schemas/CreateSubscription"
         required: true
+      callbacks:
+        notifications:
+          "{$request.body#/webhook/notificationUrl}":
+            post:
+              tags:
+                - Session notifications callback
+              summary: "Session notifications callback"
+              description: |
+                Important: this endpoint is to be implemented by the API consumer.
+                The Device status server will call this endpoint whenever any device status related event occurs.
+              operationId: postNotification
+              requestBody:
+                required: true
+                content:
+                  application/cloudevents+json:
+                    schema:
+                      $ref: "#/components/schemas/CloudEvent"
+                    examples:
+                      ROAMING_STATUS:
```

Comment:
Do we want to keep these old constants, since they are not used for the type?

Reply:
I've changed the constant name to be closer to the type but I kept only the significant part;